App/screens/ProofRequest.tsx

```diff
@@ -158,7 +194,7 @@ const CredentialOffer: React.FC<Props> = ({ navigation, route }) => {
         content={
           <FlatList
             data={retrievedCredentialsDisplay}
-            keyExtractor={(item) => item.id}
+            keyExtractor={(_, index) => index.toString()}
```

Comment:
I think it is best practice to not use index for the key

Reply:
I thought about using the connection ID, but for connection-less requests I assumed this isn't available and wen't with the index. Is there a unique ID we should lean on instead of the connection ID?

Reply:
Using credential name as key, similar to the credential offer list